plugins/inputs/sqlserver/README.md

```diff
@@ -111,6 +111,18 @@ GO
   ## This setting/metric is optional and is disabled by default.
   # health_metric = false
 
+  ## Number of query retry attempts to perform. Setting this to N will run the query a total of N+1 times.   
+  # retry_count = 2
+
+  ## The amount of time to wait between query retries
+  # retry_wait_time = 500ms
+
+  ## If true, subsequent retries will wait with an exponentially increasing wait time. 
+  ## When this property is set to true, the wait time is calculated as (2 ^ retry_attempt) * (retry_wait_time), 
+  ## for 0 <= retry_attempt <= retry_count, retry_attempt++. 
+  ## When this property is false, the wait time is the same for all retries.
+  # retry_exponential_backoff = false
```

Comment:
Do you think we should support exponential backoff by default considering that is the recommendation from MSDN - https://docs.microsoft.com/en-us/azure/azure-sql/database/troubleshoot-common-errors-issues#implementing-retry-logic ?

Reply:
Thought about this a bit. I think there are some arguments to keep the default behavior as-is, and there are some arguments to change the default behavior. 

One one hand, if we introduce retry waiting, we could accidentally make the wait time longer than some users' collection intervals. Particularly for short collection intervals. Handling the default values would also complicate the code a bit.

On the other hand, it's a bit hard to argue against the suggested defaults.